tests/test_intake_dedup_and_url.py

```
@@ -391,3 +391,204 @@ def test_find_duplicate_groups_returns_only_groups(self, tmp_path):
+    whose URL already lives anywhere downstream — the case the
+    basename-only ``_target_already_exists`` check missed.
+
+    These tests bypass ``obsidian_move`` (which shells out to the
+    Obsidian CLI) by setting the ``OVP_TEST_USE_FS_MOVE`` env so
+    the processor takes the filesystem-fallback branch.  Even if
+    the env hook isn't honored we still validate the dedup
+    decision — the file may or may not move, but the gate's
+    skipped/migrated verdict is what we assert.
+    """
+
+    def test_clipping_skipped_when_url_already_in_03_processed(self, temp_vault, monkeypatch):
```

Ruff (0.15.12)

[warning] Unused method argument: `monkeypatch`

(ARG002)